src/jsonrpc/api.rs

```diff
@@ -1,9 +1,14 @@
 use crate::{
+    database::LabelItem,
```

Comment:
Meh, this is a bit ugly to make a file in the `jsonrpc` module depend on a structure from the `database` module.

Reply:
It should import it from commands that has a pub use to database.

Reply:
Oh shit that's so much simpler than what i was trying to do at the moment lol.